src/presets/vercel.ts

```diff
@@ -67,11 +67,22 @@ export const vercel = defineNitroPreset({
           funcPrefix + ".func",
           "junction"
         );
+
+        let allowQuery = key.includes("/**") ? ["url"] : undefined;
```

Comment:
Are there any docs that mention `url` is conventional or has a specific use case to be supported by default?

Reply:
As far as I understand (the `url` default was there before my change and I'm pretty new to this Vercel stuff), this is necessary for catch-all routes to work at all. Otherwise each route that ends up in the catch-all would return the same cached html. In these cases `url` param gets set here: https://github.com/unjs/nitro/blob/3a69dd70bae87d12a6d9a86691fad48b2ebb044e/src/presets/vercel.ts#L238